cmd/connector/main.go

```diff
@@ -1,5 +1,37 @@
 package main
 
+import (
+	"time"
+
+	"github.com/serverless/event-gateway-connector/httpapi"
+	"github.com/serverless/event-gateway-connector/kv"
+	"go.etcd.io/etcd/clientv3"
+	"go.uber.org/zap"
+)
+
 func main() {
-	println("Hello")
+	// logger
+	rawLogger, _ := zap.NewDevelopment()
```

Comment:
Why not `NewProduction`? I haven't used the zap logger in a couple of years, it looks like things have changed!

Reply:
`NewProduction` has level set to error, `NewDevelopment` to debug. That's the only reason why I used that. I think it's fine for now.